google_workspace_add_on/CONTRIBUTING.md

```diff
@@ -0,0 +1,62 @@
+# Contributing
+
+Thanks for contributing to the project!
+
+# Setup
+
+While inside the `google_workspace_add_on` directory.
+
+Run
+
+```
+npm install
+clasp login
```

Comment:
Can you also mentioned how to install clasp in this markdown
```
npm install -g @google/clasp
```

Reply:
Done